src/main/java/dev/jbang/catalog/TemplateProperty.java

```diff
@@ -0,0 +1,31 @@
+package dev.jbang.catalog;
+
+import com.google.gson.annotations.SerializedName;
+
+public class TemplateProperty {
+
+	private String description;
+	@SerializedName(value = "default-value")
```

Comment:
This is just my personal preference but I'd suggest simply using `default` instead of `default-value`.